setuptools/tests/test_bdist_egg.py

```diff
@@ -26,6 +27,7 @@ def setup_context(tmpdir):
 
 
 class Test:
+
```

Comment:
Is the double new line after a class decoration something that's part of the standard? I'm not familiar with it. Can you provide a reference?